swift-coverage/swift-coverage-example/sonar-project.properties

```diff
@@ -2,7 +2,7 @@ sonar.projectKey=swift-coverage-example
 sonar.projectName=swift-coverage-example
 sonar.projectVersion=1.0
 
-sonar.swift.coverage.reportPath=Coverage.report
 sonar.sources=swift-coverage-example
 
-#sonar.coverageReportPaths=sonarqube-generic-coverage.xml
\ No newline at end of file
+sonar.coverageReportPaths=sonarqube-generic-coverage.xml
+#sonar.swift.coverage.reportPaths=Coverage.report
```

Comment:
I wonder what's the point of duplicating the coverage properties here and in the readme (`sonar-scanner` command), you even swapped commented and not commented properties as indeed it's not clear which one we should keep. 

I assume you swapped as it's accurate for latest versions. 

WDYT about dropping them from here?

Reply:
Indeed, I swapped them because it's accurate for latest versions.
I'm okay with dropping them as the property is indeed already in the `sonar-scanner` command in the README.
Will do the change